res/database/Default/database_news.xml

```diff
@@ -1485,8 +1485,8 @@ Demonstrations are expected again this year.</en>
 				<en>USA - Fear of terrorist demonstrators</en>
 			</title>
 			<description>
-				<de>Die vereinigten Staaten fürchten Anschläge beim G8-Gipfel in Paris. Powell: „Wenn die Terrorcamper nicht verschwinden, werden wir nicht teilnehmen.“</de>
-				<en>The United States fear attacks at the G8 summit in Paris. Powell: "If the terror campers don't disappear, we won't participate."</en>
+				<de>Die Vereinigten Staaten fürchten Anschläge beim G8-Gipfel in Paris. Powell: „Wenn die Terrorcamper nicht verschwinden, werden wir nicht teilnehmen.“</de>
+				<en>The United States fears attacks at the G8 summit in Paris. Powell: "If the terror campers don't go away, we won't attend."</en>
```

Comment:
fear shoul be correct

Reply:
https://grammarist.com/usage/united-states/